ChangeLog.md

```diff
@@ -3,6 +3,309 @@
 Notable changes to the FreeOrion project will be documented in this file.
 
 
+## [v0.4.6] - 2016-09-10
+
+
+### Key Changes
+
+- Reworked supply mechanics
+- Various GUI improvements
+ - GUI responsiveness / FPS improvements
+ - Made GUI more configurable
+ - Lot of other small improvements to handling/display of things
+- Turn processing time improvements
+- AI improvements
+- New specials, hulls, ship parts, giving new strategic or
+fun gameplay options
+
```

Comment:
Weapon Refinements, use Geoff's text, it's a very significant change and a substantial improvement.